pkg/env/core.go

```diff
@@ -45,8 +46,9 @@ type Core interface {
 	NewLiveConfigManager(context.Context) (liveconfig.Manager, error)
 	instancemetadata.InstanceMetadata
 
-	Component() string
+	Service() string
```

Comment:
What's the difference between service and component, I see some places are using component, some are using service.